rules/macos/credential_access_suspicious_web_browser_sensitive_file_access.toml

```diff
@@ -52,24 +52,15 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where host.os.type == "macos" and event.type in ("start", "process_started") and
-  process.args :
-    (
-      "/Users/*/Library/Application Support/Google/Chrome/Default/Login Data",
-      "/Users/*/Library/Application Support/Google/Chrome/Default/Cookies",
-      "/Users/*/Library/Application Support/Google/Chrome/Profile*/Cookies",
-      "/Users/*/Library/Cookies*",
-      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/cookies.sqlite",
-      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/key*.db",
-      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/logins.json",
-      "Login Data",
-      "Cookies.binarycookies",
-      "key4.db",
-      "key3.db",
-      "logins.json",
-      "cookies.sqlite"
-    ) and 
-  not (process.name : "wordexp-helper" and process.parent.name : ("elastic-agent", "elastic-endpoint"))
+file where event.action == "open" and process.executable != null and 
+ file.name : ("cookies.sqlite", 
+              "key?.db", 
+              "logins.json", 
+              "Cookies", 
+              "Cookies.binarycookies", 
+              "Login Data") and 
+ ((process.code_signature.trusted == false or process.code_signature.exists == false) or process.name : "osascript") and 
+ not process.code_signature.signing_id : "org.mozilla.firefox"
```

Comment:
Should we specifically ignore `/Library/Elastic/Endpoint/elastic-endpoint.app/Contents/MacOS/elastic-endpoint` from `process.Ext.effective_parent.executable`? Found this in telemetry.